### ISSUE TITLE
Events inside Event Details not Sorted

### ISSUE THREAD

**Issue by edmondchui** [twinssbc/AngularJS-ResponsiveCalendar#21]
Current if the events in the eventSource is not sorted, events on the same day are not displayed in chronological orders within the event details views.

We need to add sorting to the array after this code segment (https://github.com/twinssbc/AngularJS-ResponsiveCalendar/blob/master/src/rcalendar/calendar.js#L422-L429)

````
              rows[rowIndex][dayIndex].hasEvent = true;
              eventSet = rows[rowIndex][dayIndex].events;
              if (eventSet) {
                eventSet.push(event);
              } else {
                eventSet = [];
                eventSet.push(event);
                rows[rowIndex][dayIndex].events = eventSet;
              }
````
Code to add
````
              rows[rowIndex][dayIndex].events.sort(
                function(event1, event2) {
                  return (event1.startTime.getTime() - event2.startTime.getTime());
                }
              );
````

**Comment by twinssbc** [comment]
@edmondchui fixed in 0.1.3

**Comment by edmondchui** [comment]
It works great! Thanks @twinssbc!
